src/psalm-language-server.php

```diff
@@ -9,6 +9,8 @@
 // show all errors
 error_reporting(-1);
 
+require_once __DIR__ . '/Psalm/Internal/exception_handler.php';
```

Comment:
Would it make sense to include this file via composee files-autoloading instead?